### PR TITLE
add project logo and art [SAME VERSION]

### PR DESCRIPTION
**Pull request by flynnduism**
This PR adds an /art directory containing brand assets and a readme with notes.

Signed-off-by: flynnduism dev@ronan.design

### CODE REVIEW

**Comment by bfjelds** [comment]
a couple of the SVGs didn't look quite right in the Github viewer (https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/icon/akri-icon-dark.svg and  https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/icon/akri-icon-dark.png) ... i'm not sure if this is a problem with my browser or github or the images.

File: art/logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....133459 30.54655-44.466599 52.78-58 22.23344-13.533401 47.07653-20.3 74.53-20.3 23.97345 0 44.94991 4.833285 62.93 14.5s32.38328 21.846593 43.21 36.54v-45.82h81.78v321.32h-81.78v-46.98c-10.44006 15.080075-24.84325 27.549951-43.21 37.41-18.36676 9.860049-39.53655 14.79-63.51 14.79-27.06681 0-51.71656-6.95993-73.95-20.88-22.23345-13.92007-39.82661-33.543207-52.78-58.87-12.9534-25.326793-19.43-54.423169-19.43-87.29zm252.88 1.16c0-19.720099-3.86663-36.636596-11.6-50.75-7.73338-14.113404-18.17327-24.939962-31.32-32.48-13.14674-7.540038-27.25993-11.31-42.34-11.31-15.08008 0-28.99994 3.673297-41.76 11.02-12.76007 7.346703-23.1033 18.076596-31.03 32.19-7.92671 14.113404-11.89 30.83657-11.89 50.17s3.96329 36.249927 11.89 50.75c7.9267 14.500072 18.3666 25.616628 31.32 33.35 12.95339 7.733372 26.77659 11.6 41.47 11.6 15.08007 0 29.19326-3.769962 42.34-11.31 13.14673-7.540038 23.58662-18.366596 31.32-32.48 7.73337-14.113404 11.6-31.029901 11.6-50.75zm247.37956-110.78c10.44005-17.013418 24.06991-30.353285 40.89-40.02 16.82008-9.666715 36.05656-14.5 57.71-14.5v85.26h-21.46c-25.52013 0-44.7566 5.993273-57.71 17.98s-19.43 32.866518-19.43 62.64v160.08h-81.2v-321.32h81.2z" fill="#444a6d"/></g></svg>
```
Comment (kate-goldenring) [inline art/logo.svg:1]: This is a Sonar logo

**Comment by kate-goldenring** [comment]
[logo.png](https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/logo.png) and [logo.svg](https://github.com/deislabs/akri/blob/dce6668a99f391ac20cbb3b31e99212bf281ce68/art/logo.svg) files are Sonar logos. were they accidentally included?

**Comment by bfjelds** [comment]
It still looks like the patterns aren't quite right...they don't fill the sides when I look at this one: art/icon/akri-icon-dark.svg

**Comment by flynnduism** [comment]
> It still looks like the patterns aren't quite right...they don't fill the sides when I look at this one: art/icon/akri-icon-dark.svg

Thanks @bfjelds, still a WIP to fix this. Redrawing part of the svg to avoid this distortion in the rendering - shall have updates shortly.

**Comment by flynnduism** [comment]
> logo.png and logo.svg files are Sonar logos. were they accidentally included?

Yes 🤦 thanks @kate-goldenring for spotting that. Removed those.

---

Updated the `svg` files to fix the glitchy earlier rendering.

_Before:_

Screen Shot 2020-11-25 at 4 08 17 PM

_Now:_

Screen Shot 2020-11-25 at 4 08 49 PM

**Comment by bfjelds** [comment]
Awesome!  Almost all of the images look right to me now.  Only art/logo.svg looks off to me now.

**Comment by flynnduism** [comment]
> Awesome! Almost all of the images look right to me now. Only art/logo.svg looks off to me now.

Thanks @bfjelds that's updated now

**Review by bfjelds — APPROVED** [review]

File: art/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## AKRI Color Palette:
+
+<img width="977" alt="akri-colors" src="https://user-images.githubusercontent.com/686194/100138031-4842f900-2e42-11eb-839e-0d8daeeb142c.png">
+
+---
+
+Brand only (not for text):
+
+```
+$purplebrand:   #78FFC9;
```
Comment (kate-goldenring) [inline art/README.md:35]: I think the hex for these is swapped. I haven't checked the others, though

**Comment by flynnduism** [comment]
Some follow-up adjustments to the branding to consider here:

# Logo Wordmark

Screen Shot 2021-01-12 at 11 26 39 AM

Screen Shot 2021-01-12 at 11 26 39 AM

I've had feedback that the above typeface style for `Akri` isn't a good fit; to explore a more developer-y, monospaced approach instead of the rounded, playful font originally included in this PR.
 
## Some Alternatives

Screen Shot 2021-01-11 at 10 27 10 PM

Here are some explorations of different styles for this - ranging from typical monospaced typefaces, to some that are slightly curved / chunkier / more unusual. 

## Pairing Type + Icon

I think these ones in particular stand out:

Screen Shot 2021-01-11 at 10 28 05 PM

_PT Mono is a nice, blocky one with a line thickness that works well with the line styles in the icon._ 

Screen Shot 2021-01-11 at 10 28 19 PM

_Input is one of my favorites, and looks nice in an uppercase 'stamp' style._ 

Screen Shot 2021-01-11 at 10 28 14 PM

_Major Display Mono is a very fun one, has a clean sci-fi style. Looks striking. The lines are very thin though which can be a problem at smaller sizes (no bold option)._

**Comment by flynnduism** [comment]
I'd like to get some feedback on these approaches, to figure out the best branding style to use here. Any input appreciated!

**Comment by kate-goldenring** [comment]
Thanks @flynnduism for sharing all these font options. I think I like Input Mono the best. In general, I like the log with uppercase A and the rest of the letters lowercased. I also wonder if a font without the foot on the i would look good like this one you tried out at one point:
image

Or maybe the foot on the i solidified the end of the name and its intentional? I'm definitely not familiar with the ways of branding. 

Major display is a little too creative for me.

**Comment by flynnduism** [comment]
Thanks Kate, I really like Input too (it's very [thoughtfully created](https://input.fontbureau.com/info/) for developers)!

Input in different weights:

akri-input-standard


> I also wonder if a font without the foot on the i would look good like this one you tried out at one point

Good idea. This style where the "i" has a foot is sort of a staple characteristic of the monospaced look... the closest I could find to a monospaced type style without is [source code pro](https://adobe-fonts.github.io/source-code-pro/) (from adobe). Here I've customized Input to explore that look:

akri-input-custom

**Comment by edrickwong** [comment]
I think from the alternative fonts list, Inconsolata (with the regular casing) has a nice balance between being more developer-y without going totally monospaced. I would be curious to see if there are other weightings of it, but overall I think the font itself is a good one to go with.

**Comment by kate-goldenring** [comment]
> I think from the alternative fonts list, Inconsolata (with the regular casing) has a nice balance between being more developer-y without going totally monospaced. I would be curious to see if there are other weightings of it, but overall I think the font itself is a good one to go with.

Good point, I like how the sharpness of the A of Inconsolata works with the logo. Inconsolata works for me, too.

**Comment by flynnduism** [comment]
Ok awesome. Here's a comparison of Inconsolata weights:

akri-inconsolata-weights

My favorite is bold, where the line weight feels like a good match for the icon, but the letters still have fairly clean corners.

**Comment by romoh** [comment]
> Ok awesome. Here's a comparison of Inconsolata weights:
> 
> akri-inconsolata-weights
> 
> My favorite is bold, where the line weight feels like a good match for the icon, but the letters still have fairly clean corners.

+1 on bold.
The line weight matches the icon lines and pairs well.

**Comment by flynnduism** [comment]
This PR has been updated so that the brand assets and readme notes are now all using Inconsolata Bold. Ready for review.

akri-logos

File: art/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# AKRI Logo & Assets
```
Comment (kate-goldenring) [inline art/README.md:1]: "Akri" is foremost a proper noun -- means edge in Greek -- and secondly also conveniently and acronym (AKRI). Certainly confusing but possibly twice the fun. Could you update the documentation here to use "Akri"?

File: art/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Brand only (not for text):
+
+```
+$purplebrand:   #3E446F;
+$greenbrand:    #78FFC9;
+$greenshade:    #8CCFCE;
+$purpleshade:   #524886;
+```
+
```
Comment (kate-goldenring) [inline art/README.md:43]: Should the hex for the accent palette be added here?

File: art/README.md (hunk 3 above)
Comment (flynnduism) [reply art/README.md:1]: Thanks @kate-goldenring I've corrected the casing in the readme.

File: art/README.md (hunk 4 above)
Comment (flynnduism) [reply art/README.md:43]: Good call - added these in!

**Review by kate-goldenring — APPROVED** [review]

**Comment by romoh** [comment]
> This PR has been updated so that the brand assets and readme notes are now all using Inconsolata Bold. Ready for review.
> 
> akri-logos

@flynnduism : A minor comment, are the colors used in the dark version the same as the light version? The colors of the two sides of the cube in the dark version seem to be almost the same (greenish). they appear to have less contrast, or is that the magic of how our eyes handle change in the background?

**Comment by flynnduism** [comment]
> > This PR has been updated so that the brand assets and readme notes are now all using Inconsolata Bold. Ready for review.
> > 
> > akri-logos
> 
> @flynnduism : A minor comment, are the colors used in the dark and light versions the same? The colors of the two sides of the cube in the dark version looks very similar (greenish). they appear to have less contrast, or is that the magic of how our eyes handle change in the background?

@romoh good eyes! Yes this is intentional, the colors have variants depending on whether the logo is in a light setting or a dark setting, to ensure contrast and legibility ar maintained.

Hence this PR contains two sets of assets to be used accordingly for light and dark modes, the idea being that the branding has [adaptive colors](https://medium.com/eightshapes-llc/light-dark-9f8ea42c9081) because how we perceive color changes depending on the light/dark context.

**Review by kate-goldenring — APPROVED** [review]

**Comment by flynnduism** [comment]
Woo hoo, thanks @kate-goldenring!

**Pull request merged**